presentations/async-await-intro/slides.adoc

```diff
@@ -74,10 +62,6 @@ include::./1.rs[]
 
 == Asynchrony in Rust
 
-* [Graph of the example as execution flow]
-
-== Asynchrony in Rust
-
```

Comment:
Can we add the execution flow example here? I found with here:

https://nikomatsakis.github.io/lsrtm-2022-05/#4

Reply:
I needed up using sequence diagrams to show the difference in execution between Rust and JS.

The example turned out long, and I don't know if we want to keep it in into slide deck.